### PR TITLE
Add http-get check

### PR DESCRIPTION
**Pull request by thoward**
This adds a `http-get` check and modifies the `README` to reflect this.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -230,6 +232,62 @@ http-json OK:  The value 200 found at .status matched with expression "< 300" an
...
+
+Flags:
+  -H, --header strings           Additional header(s) to send in check request
+  -h, --help                     help for http-get
+  -i, --insecure-skip-verify     Skip TLS certificate verification (not recommended!)
+  -C, --mtls-cert-file string    Certificate file for mutual TLS auth in PEM format
+  -K, --mtls-key-file string     Key file for mutual TLS auth in PEM format
+  -T, --timeout int              Request timeout in seconds (default 15)
+  -t, --trusted-ca-file string   TLS CA certificate bundle in PEM format
+  -u, --url string               URL to test (default "http://localhost:80/")
```
Comment (jspaleta) [inline README.md:259]: change description  so `URL to get`?

File: README.md [hunk 2]
```diff
@@ -299,6 +357,28 @@ spec:
...
+api_version: core/v2
+metadata:
+  name: prometheus-node-exporter
+  namespace: default
+spec:
+  command: http-get --url http://localhost:9100
+  subscriptions:
+  - system
+  runtime_assets:
+  - sensu/http-checks:0.6.0
```
Comment (jspaleta) [inline README.md:374]: gonna need to update this runtime asset version example in the readme to match the version we expect to tag that introduces the new executable.  Small thing, but has to be changed after merge but before tagging

File: cmd/http-get/main.go [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+	}
+
+	options = []*sensu.PluginConfigOption{
+		{
+			Path:      "url",
+			Env:       "CHECK_URL",
+			Argument:  "url",
+			Shorthand: "u",
+			Default:   "http://localhost:80/",
+			Usage:     "URL to test",
```
Comment (jspaleta) [inline cmd/http-get/main.go:50]: change description to "URL to get"?

**Review by jspaleta — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (jspaleta) [reply README.md:374]: looks like this is correct expected version

**Comment by thoward** [comment]
Fixed language in help text. Merging...

**Pull request merged**